src/System.IO/ref/System.IO.cs

```diff
@@ -162,13 +170,17 @@ public abstract partial class Stream : System.IDisposable
         public abstract long Position { get; set; }
         public virtual int ReadTimeout { get { return default(int); } set { } }
         public virtual int WriteTimeout { get { return default(int); } set { } }
+        public virtual System.IAsyncResult BeginRead(byte[] buffer, int offset, int count, System.AsyncCallback callback, object state) { return default(System.IAsyncResult); }
```

Comment:
A few of us chatted offline about this briefly. I'm concerned that by hiding it, folks writing custom seems won't override it. If we change the base behavior to delegate to the XxAsync methods, that's less of an issue, but that would need to be done on all platforms, and it wouldn't apply to existing versions of the full framework. cc'ing a few folks that may have opinions: @weshaggard, @KrzysztofCwalina, @terr